cmd/oras/login.go

```diff
@@ -60,20 +79,28 @@ Example - Login with insecure registry from command line:
 
 	cmd.Flags().BoolVarP(&opts.debug, "debug", "d", false, "debug mode")
 	cmd.Flags().StringArrayVarP(&opts.configs, "config", "c", nil, "auth config path")
+	cmd.Flags().StringVarP(&opts.credType, "cred-type", "t", "docker", "type of the saved credential")
 	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "registry username")
 	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "registry password or identity token")
 	cmd.Flags().BoolVarP(&opts.fromStdin, "password-stdin", "", false, "read password or identity token from stdin")
 	cmd.Flags().BoolVarP(&opts.insecure, "insecure", "k", false, "allow connections to SSL registry without certs")
+	cmd.Flags().BoolVarP(&opts.insecure, "plain-http", "", false, "allow connectsion to registry without SSL")
```

Comment:
It would be great if the flag name was more meaningful. Flags that directly weakens existing security controls should name them, so my suggestion would be: `--insecure-allow-plain-http`.

Reply:
Added `allow` in the full parameter name and did some other corrections. Added insecure in the flag description to shorten the flag name.

Reply:
Since this breaking change impact all commands, let's create an issue and address it separately.

/cc @pjbgf @qweeah

Reply:
reverted the chanage and created a issue for that